### ISSUE TITLE
Chapter Review (Richard): Swift for Tensorflow

### ISSUE THREAD

**Issue by huan** [snowkylin/tensorflow-handbook#20]
### Reviews from @rxwei

- [ ] The expression `n = 'string'` is not grammatical because single quotes are not valid. Should be double-quotes.
- [ ] There should be a whitespace after every colon in Swift code.
- [ ] I'd suggest replacing the use of the bullet operator with `matmul`. The bullet operator is highly subject to change.
- [ ] To make it more familiar to TF / Keras developers, you can replace struct MLP with Sequential.
- [ ] The MNIST repo could use some more Swift idioms. For example, Swift APIs do not use the verb "get" in method names. Methods and functions should be named according to their side effects. Also, argument labels should form a phrase with function/method base names. You can find more info in Swift API Design Guidelines: <https://swift.org/documentation/api-design-guidelines/>.
- [ ] Some code blocks are not syntax-highlighted.
- [ ] There are occurrences of "Tehsorflow". They should be corrected to "TensorFlow"
- [ ] "Swiftc" -> "swiftc"
- [ ] "能够检查Tensor维度是否匹配" is unfortunately no longer true.
- [ ] "Chris Lattner 在 Google Brain 工作，专门从事深度学习的研究“ should be revised to "Chris Lattner 在 Google Branin 工作，专门从事深度学习系统的开发”。Chris actually does no research at all.
- [ ] Typo "S4TS"
